src/rngToolsUi/gen3/rstid/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import { Stepper } from "./stepper";
+import { RsTidSidGenerator } from "./rstid";
+import { Button, Flex } from "~/components";
+
+const items = [
+  {
+    title: "Find Target TID",
+    //   subTitle: "00:00:05",
+    status: "process",
+    //   description: "Generate TID and possible SIDs.",
+  },
+  {
+    title: "RNG TID",
+    //   subTitle: "00:01:02",
+    status: "process",
+    //   description: "RNG for a shiny starter.",
+  },
+];
+
+export const RsTid = () => {
+  const [current, setCurrent] = React.useState(0);
+  const [targetSet, setTarget] = React.useState(false);
+
+  const next = () => {
+    setCurrent(current + 1);
+  };
+
+  const prev = () => {
+    setCurrent(current - 1);
+  };
+
+  const onChange = (value: number) => {
+    setCurrent(value);
+  };
+
+  const handleTarget = (advance, value) => {
+    console.log(advance, value);
+    setTarget(true);
+    return advance;
+  };
+
+  const NextButton = () => (
+    <Flex justify="end">
+      <Button
+        trackerId="test"
+        type="primary"
+        size="middle"
+        onClick={() => next()}
```

Comment:
This is a passthrough we can remove:
```suggestion
        onClick={next}
```